### ISSUE TITLE
Branding Moodboard

### ISSUE THREAD

**Issue by GarthDB** [goatslacker/alt#205]
After chatting with @goatslacker I thought it would be a good idea to think through branding a little bit.

Let's start with some moodboarding. Feel free to post screenshots of anything that could be used for inspiration for branding/site design.

**Comment by GarthDB** [comment]
[Grunt](http://gruntjs.com/)
![image](https://cloud.githubusercontent.com/assets/125516/7329166/b97306ee-ea99-11e4-9a40-2deba64139ab.png)

**Comment by jslauthor** [comment]
![image](https://cloud.githubusercontent.com/assets/224192/7330353/fa4cacbc-eaa1-11e4-99bf-4f654248ce74.png)

**Comment by mehcode** [comment]
http://www.gomoodboard.com/boards/hSVsi0qL/share

**Comment by GarthDB** [comment]
@mehcode So what elements do you like from those?

**Comment by mehcode** [comment]
Wasn't looking at elements -- just look and feel / tone. Think we should go with a more relaxed professionalism.

http://images.huffingtonpost.com/2013-01-20-Color_Emotion_Guide22.png

**Comment by GarthDB** [comment]
@mehcode you're right, I shouldn't have said elements, I mean aspects. What aspects do you like?

So you feel that alt should be seen as a relaxed professional project?

**Comment by mehcode** [comment]
Correct. I'd want the appearance (of at least the front page) to be able to satisfy curiously titled person A that this is a _real_ project. React's homepage doesn't meet that -- it gives off a "this is someone's off-time project" -- which isn't a problem at all because it has huge names like facebook behind it.

Some examples of this (lots of key words like "scale" and "next generation" help):

http://sproutcore.com/
http://emberjs.com/
https://www.djangoproject.com/ 
http://getbootstrap.com/
https://www.docker.com/

Notice the trend of appealing to management above the fold and then below we start getting to programmers.

**Comment by ziflex** [comment]
But what about logo ? I think that's what should start from.

**Comment by goatslacker** [comment]
current logo for reference:

https://github.com/goatslacker/alt/blob/master/web/assets/alt.png

**Comment by jslauthor** [comment]
@goatslacker does alt stand for alternative? ctrl-alt? some meaning behind the name would be helpful.

**Comment by goatslacker** [comment]
It's an acronym [like npm](https://github.com/npm/npm-expansions).

**Comment by GarthDB** [comment]
@ziflex Yes we should start with a logo. Mood boarding is a great way to start the discussion about look and feel, as well as messaging (going to @jslauthor's comment), but feel free to dive into specifics on the logo if you feel inspired.

**Comment by jslauthor** [comment]
Hah, what's it stand for?

**Comment by goatslacker** [comment]
Nothing and everything :)

It's just alt

**Comment by jslauthor** [comment]
So, it's an acronym that doesn't expand to more words? Not sure that's an
acronym Josh. :)

**Comment by mehcode** [comment]
**A**lternative **L**inear **T**ransition

**Comment by goatslacker** [comment]
^  there you go!

**Comment by GarthDB** [comment]
Accept Less Tyranny
